package.json

```diff
@@ -4,14 +4,24 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"WARNING: no test specified; Change 'exit 0' to 'exit 1' at the beggining of configuring this!\" && exit 0"
+    "test": "jest"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/mongodb": "^3.5.27",
```

Comment:
Do we still use this types? If no - remove it, if yes, types should be stored in devDependencies

Reply:
Concept change after last call, i will delete this library.